.gitmodules

```diff
@@ -308,3 +308,7 @@
 	url = https://github.com/asriot/ASR595X_Freertos.git
 	branch = matter
 	platforms = asr
+[submodule "perfetto"]
+	path = third_party/perfetto/repo
+	url = https://github.com/google/perfetto.git
+	branch = master
```

Comment:
This is relevant to all platforms?

Reply:
Limited it to linux only for now (I imagine later we can do mac and android)